core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java

```diff
@@ -175,8 +179,8 @@ public static boolean isValidProteinChange(String proteinChange)
 		boolean invalid = proteinChange == null ||
 		                  proteinChange.length() == 0 ||
 		                  proteinChange.equalsIgnoreCase("NULL") ||
-		                  proteinChange.equalsIgnoreCase(NOT_AVAILABLE);
-
+		                  proteinChange.equalsIgnoreCase(NOT_AVAILABLE.get(0)) ||
```

Comment:
@sandertan this still hasn't been updated. The solution @pieterlukasse suggested looks good, I for some reason thought I was looking at python when I wrote my comment :-)